RA_Integration/RA_MemManager.cpp

```diff
@@ -50,7 +50,7 @@ void MemManager::AddMemoryBank( size_t nBankID, _RAMByteReadFn* pReader, _RAMByt
 	g_MemoryDialog.AddBank( nBankID );
 }
 
-void MemManager::ResetAll(ComparisonVariableSize nNewVarSize)
+void MemManager::ResetAll(ComparisonVariableSize nNewVarSize, ByteAddress start, ByteAddress end)
```

Comment:
ResetAll now requires begin and end addresses. Each of the processing loops was modified to only look between the specified values.

Reply:
Additionally, the loops now use a pointer into the array instead of constantly evaluating the index.